Source/WebCore/rendering/RenderLayer.cpp

```diff
@@ -4221,6 +4227,16 @@ bool RenderLayer::participatesInPreserve3D() const
     return ancestorLayerIsDOMParent(parent()) && parent()->preserves3D() && (transform() || renderer().style().backfaceVisibility() == BackfaceVisibility::Hidden || preserves3D());
 }
 
+void RenderLayer::setSnapshottedScrollOffsetForAnchorPositioning(LayoutSize offset)
+{
+    if (m_snapshottedScrollOffsetForAnchorPositioning == offset)
+        return;
+
+    // FIXME: Scroll offset should be adjusted in the scrolling tree so layers stay exactly in sync.
+    m_snapshottedScrollOffsetForAnchorPositioning = offset;
+    updateTransform();
```

Comment:
Do we need to eagerly call `updateTransform()` here? We already call it twice per layout (once from style change, once from post-layout).

Reply:
Yes, there is nothing that guarantees it is updated by someone else.